src/components/ProgressBar/component.js

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
```

Comment:
hey @joshmadewell just curious to know who does `import * as React from 'react'` different from `import React from 'react'`?